eccehomo/test_settings.py

```diff
@@ -0,0 +1,4 @@
+import os
+
+TEST_MEDIA_ROOT = os.path.join(os.getcwd(), 'test_media')
+filename = os.path.join(TEST_MEDIA_ROOT, 'test_image.jpg')
```

Comment:
I have removed the settings file and create the image in the `setUpClass` for the `test_app` tests. So the `filename` variable is now contained to that file.